src/hooks/useStep.tsx

```diff
@@ -65,12 +65,12 @@ export const useStep = () => {
     }
   };
 
-  const currentStepId = pathname ? parseInt(getLastPartOfPath(pathname)) : FIRST_STEP.id;
+  const currentStepId = pathname ? getLastPartOfPath(pathname) : FIRST_STEP.id;
 
   const chainStepMap: Record<string, Step> =
     chainType === ChainType.Rollup ? RollupStepMap : AnyTrustStepMap;
 
-  const findStepById = (id: number): Step | undefined => {
+  const findStepById = (id: string): Step | undefined => {
```

Comment:
NIT: Maybe we should reconsider how we store those steps. Array of ids for the order and map for easy access?